conn.go

```diff
@@ -36,13 +38,13 @@ func (c *conn) IsClosed() bool {
 
 // OpenStream creates a new stream.
 func (c *conn) OpenStream() (mux.MuxedStream, error) {
-	qstr, err := c.sess.OpenStreamSync()
+	qstr, err := c.sess.OpenStreamSync(context.Background())
```

Comment:
We should probably set a timeout. 5s?

(ideally, the context would be passed through but that's better than nothing)

Reply:
That might be a fix for #57. Let's do that in a separate PR.